dbcrossbarlib/src/drivers/dbcrossbar_ts/shopify.ts

```diff
@@ -0,0 +1,329 @@
+// This is a Shopify REST schema that we built by reading the docs.
+
+// A decimal value represented as a string for accuracy.
+type Decimal = string;
+
+// A type that we still need to look up.
+//type TODO = string;
+
+// https://shopify.dev/docs/admin-api/rest/reference/orders/order?api[version]=2020-04
+interface Order {
```

Comment:
um, wow

Reply:
Yeah, it's pretty complex. There's a reason I built a new schema parser just for this. :-)

This isn't the final schema. We'll need to add lots of `| null`, etc.